src/core/client/stream/tabs/Comments/Stream/UnansweredCommentsTab/UnansweredCommentsTabContainer.tsx

```diff
@@ -183,7 +179,7 @@ export const UnansweredCommentsTabContainer: FunctionComponent<Props> = (
                     viewer={props.viewer}
                     comment={comment}
                     story={props.story}
-                    onRemoveAnswered={onRemoveAnswered}
+                    showRemoveAnswered={true}
```

Comment:
Boolean properties don't need a value!

```suggestion
                    showRemoveAnswered
```